### DISCUSSION TITLE
kitten choose-fonts not showing in my kitty terminal

### DISCUSSION THREAD

**Discussion by ciscohack** [kovidgoyal/kitty#7627] [General]
Hello Experts,

I don't see new kitten choose-fonts in my kitten list any idea why? when I ran the command  I got error

kitten choose-fonts

<img width="1050" alt="image" src="https://github.com/user-attachments/assets/f5d4875e-dd1f-435a-9018-41a0eb2568e3">

**Comment by kovidgoyal** [comment]
Because it isnt released yet.

**Reply by ciscohack** [reply]
oh okay thanks

**Reply by AvicennaJr** [reply]
Which version will it be released on as I am facing the same issue?  Currently on kitten 0.35.2

**Reply by ciscohack** [reply]
@AvicennaJr kitty 0.36 has this feature but keep in mind if you upgrade this version then might be your favorite font not work at it stricktly check mono font

**Reply by AvicennaJr** [reply]
@ciscohack thanks a lot. It is very easy to edit the fonts (and everything else) on the config file. A minor enhancement on the docs could be to add the supported version number to run the command `kitten choose-fonts`.

**Comment by forrestblade** [comment]
I don't understand how to add dank mono into choose-font

**Comment by UnoYakshi** [comment]
This command helped me to install fonts on kitty 0.39.0: `kitty +list-fonts --psnames`.
Found it here: https://github.com/kovidgoyal/kitty/discussions/4380#discussioncomment-1854089.

![image](https://github.com/user-attachments/assets/92ac4a0f-5fc0-4926-8fdf-5bdb45b8727d)
